src/generalComponents/Selector.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const SelectorWrapper = styled.div`
+display: block;
+width: 23.1px;
+height: 19.8px;
+color: #FFF;`
+
+const OptionSelector = styled.div`
+width: 23.1px;
+height: 3px;
+background: #FFF;
+color: #FFF;
+margin: 4px 0px 4px 5px;
+`
+const OpltionList = styled.ul`
+list-style: none;
+top: 30px;
+left: 0px;
+color: #FFF;
+background: red;
+display: ${props => {
+    console.log(props.display);
+    props.display
+}};
+`
+const SetOption = styled.li`
+cursor: pointer;
+color: white;
+`
+const Selector = () => {
```

Comment:
Nie widziałem tego nigdzie na froncie apki, domyślam się że to początek mobilnej nawigacji. Znów - nie trzeba jej pisać jeszcze raz. Można użyć komponentu z listą przycisków nawigacji, a następnie za pomocą propsa np. "mobile" zmienić style